lib/msf/base.rb

```diff
@@ -1,5 +1,3 @@
-# -*- coding: binary -*-
```

Comment:
Sanity question, is this needed still?

Reply:
it shouldn't be, it only seems to affect any strings defined in the file and there aren't any :smile: 
weren't any before so it was probably a copy/paste job

Reply:
That's my understanding too, just calling it out for visibility - as there seemed to be a couple of historical bulk changes to add this to every file:
https://github.com/rapid7/metasploit-framework/commit/d656e3185ff80b7264c30e994b2299a039e907a2

I imagine there was a solid reason for it that, but the commit doesn't have any further details. It could've just been the easiest to automate, adding any missing binary comment regardless of the file's content